test/programs/argument-id/schema.MyObject.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "someSchemaId",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "ReferenceType": {
+            "properties": {
+                "reference": {
+                    "enum": [
+                        true
+                    ],
+                    "type": "boolean"
+                }
+            },
+            "required": [
+                "reference"
+            ],
+            "type": "object"
+        }
+    },
+    "properties": {
+        "referenceType": {
+            "$ref": "someSchemaId#/definitions/ReferenceType"
```

Comment:
Doesn't someSchemaId have to be a URL? I'd like this to be a somewhat plausible example.